spec/pods.md

```diff
@@ -179,3 +180,7 @@ JSON Schema for the Pod Manifest, conforming to [RFC4627](https://tools.ietf.org
 * **ports** (list of objects, optional) list of ports that SHOULD be exposed on the host.
     * **name** (string, required, restricted to the [AC Name](#ac-name-type) formatting) name of the port to be exposed on the host. This field is a key referencing by name ports specified in the Image Manifest(s) of the app(s) within this Pod Manifest; consequently, port names MUST be unique among apps within a pod.
     * **hostPort** (integer, required) port number on the host that will be mapped to the application port.
+* **exitPolicy** (string, optional) a string that specify the exit policy of the pod, if left empty, then it's up to ACE to choose the default behavior. Valid values are:
```

Comment:
First: Kubernetes is assuming "untilAll", and we resisted adding this for lack of really concrete use-cases.  My instinct is that it SOUNDS cool, but isn't that useful IRL.  As far as I know we have no such equivalent internally.  If any app container exits with failure, we know the pod is doomed to fail, but we let the other containers finish.

But I'm going to assume you have a concrete set of use-cases that justify this (you should write them down in this PR description) or else you would not be adding hypothetical complexity.

I just went to refresh on the state of the spec, and I realize there is no (what kubernetes calls) `restartPolicy`.  Is this supposed to be an analog of that?  I think it's interesting to contrast the approaches.

Kubernetes defines:

* RestartAlways: Always restart app containers, regardless of exit code.  The pod can only terminate in Failure if the runtime decides that it is not viable (hardware failure, machine drain, etc).
* RestartOnFailure: Restart containers if and only if they exited with a non-zero code.  The pod's terminal state is the worst-of any container's terminal state.
* RestartNever: Never intentionally restart containers.  The pod's terminal state is the worst-of any container's terminal state.

Superficially kube's `RestartAlways` feels the same as `untilAll` here.  But here's the rub - the definition of untilAll doesn't actually say anything about _restart_.  Is that part of the policy here or is that governed somewhere else that I am not seeing?

I'll not write much more now, because I have asked enough questions that I am probably attacking a straw man.

From a functional POV I think the concepts that matter to a user are "when does my container get restarted?" and "what does that mean for the fate of my pod?", but this only answers the latter, and only partially.

From an API usability POV I think it might be clearer to express these things "in the positive".  E.g. I think a "RunPolicy" would be clearer (RunForever, RunToCompletion, RunOnce), and I sort of wish Kubernetes had done it that way.